samples/bluetooth/broadcaster_multiple/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _bluetooth-broadcaster-multiple-sample:
+
+Bluetooth: Multiple Broadcaster
+###############################
+
+Overview
+********
+
+A simple application demonstrating the Bluetooth Low Energy Broadcaster that
+uses multiple advertising sets functionality.
+
+Requirements
+************
+
+* A board with Bluetooth Low Energy support
+
+Building and Running
+********************
+
+This sample can be found under
+:zephyr_file:`samples/bluetooth/broadcaster_multiple` in the Zephyr tree.
+
+See :ref:`bluetooth samples section <bluetooth-samples>` for details.
```

Comment:
I would expect more extensive documentation describing what the sample does, and how to validate it. See for example https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hr_coded/README.html